### PR TITLE
kaniko-migration

### PR DESCRIPTION
**Pull request by gosharo**
https://github.com/Rookout/jenkins-k8s/pull/723

### CODE REVIEW

**Comment by gosharo** [comment]
[RK-18359](https://rookout.atlassian.net/browse/RK-18359)

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -18,27 +19,38 @@ podCreator.initMain(label: label) {
...
+                    if(Utilities.checkIfFileExist(this, dirPath, "Dockerfile.test"))
+                    {
+                        dockerfile = "Dockerfile.test"
+                    }
+                    else
+                    {
+                        dockerfile = "Dockerfile"
                     }
-                    TestUtilities.createTest(this, scriptToUse, jobName, pwd() + "/${testsNames[i]}", "Jenkinsfile", testFolder)
+                    podCreator.initKaniko(branch: env.GIT_BRANCH,
```
Comment (ElDuderinos) [inline .jenkins/Jenkinsfile:41]: Won't we a have a problem with the slash replacement in branches with slashes?

**Review by ElDuderinos — CHANGES_REQUESTED** [review]

File: .jenkins/Jenkinsfile (hunk 1 above)
Comment (gosharo) [reply .jenkins/Jenkinsfile:41]: We didn't remove the line, it's just went down

Comment (gosharo) [reply .jenkins/Jenkinsfile:41]: We will, on it

**Review by alexeygutkin — APPROVED** [review]

**Pull request merged**